fhttp/http.go

```diff
@@ -57,14 +57,19 @@ var (
 )
 
 // NewHTTPOptions creates and initialize a HTTPOptions object.
+// It replaces plain % to %25 in the url. If you already have properly
+// escaped URLs use o.URL = to set it.
 func NewHTTPOptions(url string) *HTTPOptions {
 	h := HTTPOptions{}
 	return h.Init(url)
 }
 
 // Init initializes the headers in an HTTPOptions (User-Agent).
+// It replaces plain % to %25 in the url. If you already have properly
+// escaped URLs use o.URL = to set it.
 func (h *HTTPOptions) Init(url string) *HTTPOptions {
-	h.URL = url
+	// unescape then rescape % to %25 (so if it was already %25 it stays)
+	h.URL = strings.Replace(strings.Replace(url, "%25", "%", -1), "%", "%25", -1)
```

Comment:
do we want something here like: `url.QueryEscape()` and `url.QueryUnescape()`  or is the `%` special here?

Reply:
query escape would mangle / and other characters, % is somewhat special/hacky yes

I'm not sure what to do cleanly here as the url with a raw % is invalid, I'm just making it work despite being invalid so it looks pretty in the UI text field (I think that's also why I didn't have "%" the first time around for the status weights PRs)